docs/tutorials/up12_initial_configuration.rst

```diff
@@ -0,0 +1,44 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _up12_initial_configuration:
+
+.. |propeller_name| replace:: UP12
+.. |motor_name| replace:: Vertiq 2306
+
+****************************************************
+Setting up the |motor_name| for the |propeller_name|
+****************************************************
+
+For the |propeller_name| to create control torque, the |motor_name| must have the appropriate firmware and settings applied. You can download the pulsing firmware for the |motor_name| on the `underactuated propeller page <https://www.vertiq.co/up12>`_. To flash the downloaded firmware use the instructions in :ref:`updating_firmware`.
+
+Once the firmware is flashed, connect the module to IQ Control Center and go to the ‘General’ tab. Download the :download:`UP12 Custom Defaults File<../_static/defaults_files/custom_defaults_pulsing_12x4.json>`. Use the ‘Import’ button to load the defaults file into IQ Control Center.
```

Comment:
Update this to include the thermal convection term that we just added?

Reply:
Is there a place where we upload default files to git or something? I'd like this is be version controlled. Also if the param doesn't exist in control center or the motor will it crash?

Reply:
@vertiq-jordan added conv term to defaults and tested